ui/src/components/songsView/SongsButtonsCard.tsx

```diff
@@ -38,8 +41,9 @@ const SongsButtonsCard = ({ song, userView = false, userHeader = false }: SongsB
   const [split, setSplit] = useState(1);
   const theme = useTheme();
   const [showMobileInfo, setShowMobileInfo] = useState(false);
-
-  const isDesktop = useMediaQuery('(min-width:768px)');
+  const [showSplitSnackbar, setShowSplitSnackbar] = useState(false);
+  const isSmallScreen = useMediaQuery('(max-width:600px)');
+  const isDesktop = useMediaQuery('(min-width:920px)');
```

Comment:
[later]: I saw a lot of this useMediaQuery with different width for different pages. Would be better if we can define a file for all the width i.e.
```
MOBILE_WIDTH = "600px"
TABLET_WIDTH = "920px"
```